packages/flutter/lib/src/cupertino/segmented_control.dart

```diff
@@ -82,6 +77,10 @@ class SegmentedControl<T> extends StatefulWidget {
     @required this.children,
     @required this.onValueChanged,
     this.groupValue,
+    this.unselectedColor = CupertinoColors.white,
```

Comment:
Maybe briefly mention the customization options in the doc comment above?

Reply:
Added a couple sentences to main doc comment, but not constructor (let me know if I should add something to constructor comment as well?)

Reply:
If there are parameters that should not be null, assert them